docs/user/alerting/rule-types.asciidoc

```diff
@@ -47,7 +50,7 @@ Domain rules are registered by *Observability*, *Security*, <<maps, Maps>> and <
 | Run an {es} query to determine if any documents are currently contained in any boundaries from a specified boundary index and generate alerts when a rule's conditions are met.
 
 | {ml-docs}/ml-configuring-alerts.html[{ml-cap} rules] beta:[]
-| Run scheduled checks on an anomaly detection job to detect anomalies with certain conditions. If an anomaly meets the conditions, an alert is created and the associated action is triggered.
+| Run scheduled checks on an {anomaly-job} to detect anomalies with certain conditions. If an anomaly meets the conditions, an alert is created and the associated action is triggered.
```

Comment:
When I view the output, the hover help is cut off. Perhaps it would work better within the table if it's moved here?
```suggestion
| beta:[] Run scheduled checks on an {anomaly-job} to detect anomalies with certain conditions. If an anomaly meets the conditions, an alert is created and the associated action is triggered.
```